### ISSUE TITLE
Is Cirrus down?

### ISSUE THREAD

**Issue by cjdb** [cirruslabs/cirrus-ci-docs#272]
We're running CI for https://github.com/CaseyCarter/cmcstl2/pull/291, but [jobs 1 and 2 are hanging][hanging] (they shouldn't take this long to complete and I can't access the logs).

Is this a server-side problem or a client-side problem?

[hanging]: https://cirrus-ci.com/build/5077460927381504

**Comment by RDIL** [comment]
Most likely an issue within the task.  I would try cancelling and rerunning.

**Comment by RDIL** [comment]
Works fine for me, I think it is just your task stuck hanging.  Restarting it is probably your best option.

**Comment by fkorotkov** [comment]
Please email support@cirruslabs.org next time to get faster response.

I see some logs now. You didn't see the logs probably because there were no new logs and Cirrus only shows new logs in real-time (there is no buffer at the moment). Please follow #138 for updates on this UX issue.

Seems the issue started happening after this commit https://github.com/CaseyCarter/cmcstl2/commit/afdc1691aff91cbd8112de9812e789653884960d and it looks like to be very reproducible even though it passes on version 8 of the compiler. 🤔 Do you think it might be related? Seem it always hangs on the following file:

```
[337/339] : && /usr/bin/g++-7  -O3 -DNDEBUG   test/view/CMakeFiles/span.dir/span.cpp.o  -o test/view/span   && :
```

**Comment by cjdb** [comment]
Yes, after @RDIL helped me out, I did a bit of digging, and realised that it's definitely an image issue -- apparently Bionic Beaver doesn't ship with GCC 7.4 (despite that having been released months before the image was built).

I've kicked off a Disco Dingo build now, so that will (hopefully) fix the problem when I push it in the next hour or so.

> Please email support@cirruslabs.org next time to get faster response.

Thanks for the email address!
